Dockerfile

```diff
@@ -48,10 +29,28 @@ RUN apt-get install -y apt-transport-https software-properties-common && \
     usermod -a -G 0 kbase && \
     usermod -a -G 999 kbase
 
-USER kbase:999
+# The htcondor package tries an interactive config, set DEBIAN_FRONTEND to
+# noninteractive in order to prevent that
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get install -y htcondor zile vim libgomp1 && \
+    mkdir /scratch && \
+    cd /tmp && \
+    wget http://submit-3.batlab.org/nmi-runs/condorauto/2018/03/condorauto_submit-3.batlab.org_1520871025_1539075/userdir/nmi:x86_64_Debian9/results.tar.gz && \
```

Comment:
What is this?

Reply:
Not sure, this was already in the debian image that Steve created

Reply:
Also I separated out the steps for easier debugging